pkg/crc/ssh/ssh.go

```diff
@@ -92,3 +95,15 @@ err     : %w\n`, command, err)
 
 	return string(stdout), string(stderr), nil
 }
+
+func WaitForSSH(ctx context.Context, sshRunner *Runner) error {
+	checkSSHConnectivity := func() error {
```

Comment:
Should it be `sshRunner.WaitForConnectivity(ctx)` rather than `WaitForSSH(ctx, sshRunner)`? Maybe the timeout should be passed as a parameter?

Reply:
@cfergeau will check that, I just focused on the moving it to this package.

Reply:
Nit: It's in the `ssh` package, so the `SSH` mention in `WaitForSSHConnectivity` is redundant.